src/audio-manager.ts

```diff
@@ -331,41 +373,81 @@ export class AudioManager {
      *
      * @warning This does not work for one-shots!
      *
-     * @param sourceId Identifier of the audio source that should be stopped.
-     * @param playId Optional parameter that specifies the exact audio that should be stopped.
-     * If not provided, all playing instances of the given sourceId will be stopped.
+     * @param playId Specifies the exact audio that should be stopped.
+     * @param sourceId Optional paramter, left for backwards compatablity. Doesnt do anything
      *
      * @note Obtain the playId from the play() method.
      * @see play
+     */ // @todo jonathan: Now it works like it always should have. Is this fine for backwards compat?
```

Comment:
Switching the parameters is absolutely breaking the API :)
You could also consider it a bug, though, and question is if people used this function 🤔

Reply:
So how should i handle this now? :fearful: